src/components/ClockFace.js

```diff
@@ -1,13 +1,12 @@
 import React, { useState, useEffect, createContext } from "react";
 import classnames from "classnames";
 import SettingsModal from "./SettingsModal";
-
-// variables outside components are the ones that don"t change, or as helper functions exposure
+import "./ClockFace.scss";
 
 export const ModalContext = createContext();
 
 const ClockFace = () => {
-  const [sessions, setSessions] = useState([65, 5]);
+  const [sessions, setSessions] = useState([1500, 300, 1500, 300]);
```

Comment:
@KevinHong913 I know you said put the sessions outside the func, but the only reason why I can think of for doing this is, if this doesn't change throughout the app. 
But sessions can be changed by the user now.